typesupply/fontMath
# Unit tests

- Move doctests to unittests in fontMath/test
- Use tox to test py27, pyp, py34, py35 (dropped py26)
- Remove Transform test that belong to fontTools (see #26)
- Add appveyor.yml

AppVeyor needs to be enabled.
Coveralls should be enabled as well.
